go.mod

```diff
@@ -6,36 +6,47 @@ require (
 	github.com/ChainSafe/chainbridge-core v1.4.2
 	github.com/binance-chain/tss-lib v0.0.0-00010101000000-000000000000
 	github.com/centrifuge/go-substrate-rpc-client v2.0.0+incompatible
```

Comment:
Can you please references to this and remove this dependecy as well?
For reference all imports that are `.../go-substrate-rpc-client` should go to the v4 version